ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -78,23 +84,22 @@ def create_catalog_source_yaml(self):
 
     def install_ocs_operator(self):
         """
-        Install OCS Opeartor
+        Install OCS/ODF Opeartor
 
         """
         self.navigate_operatorhub_page()
-
-        logger.info("Search OCS Operator")
-        self.do_send_keys(
-            self.dep_loc["search_operators"], text="OpenShift Container Storage"
-        )
-
-        logger.info("Choose OCS Version")
-        self.do_click(self.dep_loc["choose_ocs_version"], enable_screenshot=True)
-
+        self.do_send_keys(self.dep_loc["search_operators"], text=self.operator)
+        logger.info("Choose OCS/ODF Version")
+        if self.operator is OCS_OPERATOR:
+            self.do_click(self.dep_loc["choose_ocs_version"], enable_screenshot=True)
+        elif self.operator is ODF_OPERATOR:
+            self.do_click(self.dep_loc["click_odf_operator"], enable_screenshot=True)
         logger.info("Click Install OCS")
         self.do_click(self.dep_loc["click_install_ocs"], enable_screenshot=True)
+        if self.operator is ODF_OPERATOR:
+            self.do_click(self.dep_loc["enable_console_plugin"], enable_screenshot=True)
         self.do_click(self.dep_loc["click_install_ocs_page"], enable_screenshot=True)
-        self.verify_operator_succeeded(operator="OpenShift Container Storage")
+        self.verify_operator_succeeded(operator=self.operator)
 
     def install_local_storage_operator(self):
```

Comment:
Are we also planning to add MCG only deployment? Only storagesystem creation steps will vary in that case.

Reply:
I will add "MCG only" deployment on another PR..